tests/test-data/bridges/appBridges.ts

```diff
@@ -28,6 +28,7 @@ import { TestsUserBridge } from './userBridge';
 export class TestsAppBridges extends AppBridges {
     private readonly appDetails: TestsAppDetailChangesBridge;
     private readonly cmdBridge: TestsCommandBridge;
+
```

Comment:
```suggestion

```

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/149/commits/2a4e5ca641b067de0dd4d609d7f1877478e863d3